components/playlist/browser/resources/components/contextualMenu.tsx

```diff
@@ -126,27 +46,27 @@ export default function ContextualMenuAnchorButton ({
   onShowMenu,
   onDismissMenu
 }: MenuProps) {
-  const [showingMenu, setShowingMenu] = React.useState(false)
+  const menuButtonProps: ButtonMenuProps = {}
+  // Force menu widget to be closed when the anchor button is not visible. In
+  // case where it's visible, the menuButtonProps doesn't contain isOpen property,
+  // so it won't affect the behavior of the menu.
+  if (!visible) menuButtonProps.isOpen = false
```

Comment:
Does this actually do anything?

Reply:
Yeah, when mouse leaves items, the menu will be hidden. When mouse came back into the item, menu is still open without this.